Fixing postgis.yml doesn't include yum / RedHat variation #258

justinwrobel opened the pull request:
- Added common yum repo
- Updated postgresql_ext_postgis_deps for RedHat

justinwrobel commented:
I tested with the following and the playbook completed without any noticeable errors. 

Here is the test scenario and output:

**test/vars.yml**
```
postgresql_ext_install_postgis: yes
```

**Invocation**
```
vagrant up centos.local;
...
PLAY RECAP *********************************************************************
centos7.local              : ok=34   changed=19   unreachable=0    failed=0    skipped=47   rescued=0    ignored=0
```

gclough commented:
Seems like a good idea @justinwrobel .  Could you rebase it, which should also fire the new CI pipeline?

justinwrobel commented:
So I've rebased but I'm not sure why these are failing. Following is the error :

> Computed fully qualified role name of postgresql does not follow current galaxy requirements.

Sadly, I don't have a bunch of time for this as I've moved on to another project. If it is something simple I'd be glad to accommodate the change otherwise I can close this MR.
